wasm-crypto/Cargo.toml

```diff
@@ -0,0 +1,27 @@
+[package]
+name = "wasm-crypto"
+license.workspace = true
+version.workspace = true
+edition.workspace = true
+rust-version.workspace = true
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[lib]
+crate-type = ["cdylib"]
+
+[dependencies]
+crypto = { path = '../crypto' }
+serialization = { path = "../serialization" }
+
+# This crate is required for rand to work with wasm. See: https://docs.rs/getrandom/latest/getrandom/#webassembly-support
+getrandom = { version = "0.2", features = ["js"] }
+wasm-bindgen = "0.2"
+thiserror.workspace = true
+
+[dev-dependencies]
+rstest = "0.16"
+test-utils = {path = '../test-utils'}
+
+[profile.release]
+lto = true
```

Comment:
I'm curious why lto is used for this module.

Reply:
It's recommended for wasm to make the size of the binary smaller.